internal/api/client/handlers/save/save.go

```diff
@@ -0,0 +1,106 @@
+package save
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+)
+
+var (
+	ErrNotEmpty = fmt.Errorf("empty data entered")
```

Comment:
Имелось ввиду Error Empty, я думаю?
```suggestion
	ErrEmpty = fmt.Errorf("empty data entered")
```